use-the-built-in-sql-client.md

```diff
@@ -148,26 +148,50 @@ $ echo "SHOW TABLES; SELECT * FROM roaches;" | cockroach sql --user=maxroach --h
 +-----------------------+---------------+
 ~~~
 
-In these examples, we show tables and special characters printed with and without the `--pretty` flag. Note that when not using `--pretty`, table rows are printed as tab-separated values, and special characters are escaped; thus, the output is easy to parse by other programs.
+### Print with or without pretty output
+
+In these examples, we show tables and special characters printed with and without pretty output. When pretty output is enabled, tables are printed with ASCII art and special characters are not escaped for easy human consumption. When pretty output is disabled, table rows are printed as tab-separated values, and special characters are escaped; thus, the output is easy to parse by other programs.
+
+When the standard output is a terminal, pretty output is enabled by default, but you can explicitly disable it with `--pretty=false`:
 
 ~~~ shell
-# Using the --pretty flag in conjunction with --execute:
-$ cockroach sql --pretty --execute="INSERT INTO emojis VALUES ('🐥 '), ('🐢 ') RETURNING *;"
-+----+
-| a  |
-+----+
-| 🐥  |
-| 🐢  |
-+----+
-
-# Using --execute without the --pretty flag:
-$ cockroach sql --execute="INSERT INTO emojis VALUES ('🐥 '), ('🐢 ') RETURNING *;"
-2 rows
-a
-"\U0001f425 "
-"\U0001f422 "
+# Using the default pretty output:
+$ cockroach sql --pretty --execute="SELECT '🐥' AS chick, '🐢' AS turtle"
++-------+--------+
+| chick | turtle |
++-------+--------+
+| 🐥    | 🐢     |
++-------+--------+
+
+# Explicitly disabling pretty output:
+$ cockroach sql --pretty=false --execute="SELECT '🐥' AS chick, '🐢' AS turtle"
+1 row
+chick turtle
+"\U0001f425"  "\U0001f422"
 ~~~
 
+When piping output to another command or a file, the default is reversed. Pretty output is disabled by default, but you can explicitly request it with `--pretty`:
+
+~~~ shell
+# Using the default non-pretty output:
+$ cockroach sql --execute="SELECT '🐥' AS chick, '🐢' AS turtle" > out.txt
+$ cat out.txt
+1 row
+chick turtle
+"\U0001f425"  "\U0001f422"
+
+# Explicitly requesting pretty output:
+$ cockroach sql --pretty --execute="SELECT '🐥' AS chick, '🐢' AS turtle" > out.txt
+$ cat out.txt
++-------+--------+
+| chick | turtle |
++-------+--------+
+| 🐥    | 🐢     |
++-------+--------+
+~~~
+
+If `--pretty` is specified without `--execute`, it will apply to the format of any tables output in the resulting interactive SQL shell.
```

Comment:
You bet! I was going for "it will apply to the format of any tables *that are* output..." I've updated to "it will apply to the format of every table's output in the resulting..." to avoid the ambiguity and the awkward plural possessive "tables'".